sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -781,3 +782,67 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function that returns number of elements in outer Json Array.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(jsonArray) - Returns length of the jsonArray",
+  arguments = """
+    jsonArray - A JSON array is required as argument. `Analysis Exception` is thrown if any other
+    valid JSON expression is passed. `NULL` is returned in case of invalid JSON.
```

Comment:
Can you follow the other format? e.g., https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala#L292

Reply:
ok. I will make the changes